README.md

```diff
@@ -319,6 +319,17 @@ bash update_docs.sh
 pdoc --http 0.0.0.0:8080 laplace --template-dir template
 ```
 
+## Contributing
+
+Pull requests are very welcome. Please follow these guidelines:
+
+1. Use [ruff](https://github.com/astral-sh/ruff) as autoformatter. Please refer to the following [makefile](https://github.com/aleximmer/Laplace/blob/main/makefile) and run it via `make ruff`. Please note that the order of `ruff check --fix` and `ruff format` is important!
```

Comment:
Maybe add something like
1. Install via `pip install -e .[dev]` which will install `ruff` and all requirements necessary to run the tests and build the docs.